single_select_plugin/package.json

```diff
@@ -0,0 +1,120 @@
+{
+    "name": "single-select-plugin",
+    "version": "1.0.0",
+    "description": "",
+    "author": "",
+    "private": true,
+    "license": "MIT",
```

Comment:
Consider linking to the root license file instead, e.g.: `SEE LICENSE IN ../LICENSE`
See how it's done in other plugins: https://github.com/gooddata/gooddata-plugin-examples/blob/master/dashboard_description_plugin/package.json#L7